lib/kindle_highlights.rb

```diff
@@ -3,6 +3,10 @@
 require 'active_support/core_ext/object/blank'
 require 'active_support/core_ext/string/filters'
 
+module KindleHighlights
+  KINDLE_ROOT = "https://read.amazon.com"
```

Comment:
I like this idea, but I'd really like to not have any constants in this file.

What are your thoughts on moving this to the `Client` class?

Reply:
I put it on the class just because `Book` class refer the constant too.
But moving this constant to the `Client` class and transfer the `fetch_highlights_from_amazon` function to the `Client` class might be better way.

Reply:
I'd like to see that, just to keep all the constants in one spot.